locscale/__init__.py

```diff
@@ -63,8 +63,31 @@ def isVersionActive(cls):
         return cls.getActiveVersion().startswith(V0_1)
 
     @classmethod
-    def defineBinaries(cls, env):
+    def getEmanPlugin(self):
+        # --- Eman2 dependencies ---
+        try:
+            emanPlugin = Domain.importFromPlugin("eman2", "Plugin",
+                                                 doRaise=True)
+        except Exception as e:
+            print(pwutils.redStr("Eman plugin does not installed....You need to install it "
+                  "first."))
+            return None
```

Comment:
Try/Catch is not needed, `importFromPlugin` has an internal try/catch.

I would replace this peace of coda as:
```
def getEmanPlugin(self):
        # --- Eman2 dependencies ---
        return Domain.importFromPlugin("eman2", "Plugin",
                                       errorMsg="Eman plugin is not installed... "
                                                "You need to install it to run locscale.")
```
`importFromPlugin` returns `None` when plugin/module not found